### ISSUE TITLE
Support custom `_id`s

### ISSUE THREAD

**Issue by jeromegn** [sergeyksv/tingodb#2]
Hey there,

I was looking at nedb before, but this looks even better.

Now I was wondering if it would be possible to specify a custom _id when inserting a document. Right now it just throws "Invalid object key (_id)" at me.

This is useful since I'm storing Facebook user information. Their UID is unique enough for me. Sometimes I also store Google Accounts information, in that case, a email is unique enough. (this is a local database without much data)

Basically, using custom ids would simplify some of my schemas and some logic.

Keep it up!
